### PR TITLE
Add support for multi-repo on the ci webhook

### PR DESCRIPTION
**Pull request by afrittoli**
# Changes

Change the CI pipeline name for plumbing to "tekton-plumbing-ci-pipeline"
Added CEL filters to ensure we only run the plumbing pipeline on
changes to the plumbing repo.
Make it possible for several repos to use the kind label check.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: tekton/ci/eventlistener.yaml [hunk 1]
```diff
@@ -67,6 +69,8 @@ spec:
               - pull_request
         - cel:
             filter: >-
+              body.repository.full_name.startsWith('tektoncd/') &&
+              body.repository.name in ['plumbing', 'pipeline', 'triggers', 'cli', 'dashboard'] &&
```
Comment (vdemeester) [inline tekton/ci/eventlistener.yaml:73]: can we add `hub` in there ? :pray:

Comment (vdemeester) [reply tekton/ci/eventlistener.yaml:73]: (and maybe `catalog` too ?)

Comment (afrittoli) [reply tekton/ci/eventlistener.yaml:73]: done

**Pull request merged**